packages/elements-core/src/components/TryIt/Body/FormDataBody.tsx

```diff
@@ -24,61 +25,120 @@ export const FormDataBody: React.FC<FormDataBodyProps> = ({
+            const value = values[parameter.name ?? ''];
+
+            if (supportsFileUpload) {
+              return (
+                <FileUploadParameterEditor
+                  key={parameter.name}
+                  parameter={parameter}
+                  value={value instanceof File ? value : undefined}
+                  onChange={newValue =>
+                    newValue
+                      ? onChangeValues({ ...values, [parameter.name]: newValue })
+                      : onChangeValues(omit(values, parameter.name))
+                  }
+                />
+              );
+            }
 
-          if (supportsFileUpload) {
             return (
-              <FileUploadParameterEditor
+              <ParameterEditor
                 key={parameter.name}
                 parameter={parameter}
-                value={value instanceof File ? value : undefined}
-                onChange={newValue =>
-                  newValue
-                    ? onChangeValues({ ...values, [parameter.name]: newValue })
-                    : onChangeValues(omit(values, parameter.name))
+                value={typeof value === 'string' ? value : undefined}
+                onChange={value =>
+                  onChangeValues({
+                    ...values,
+                    [parameter.name]: typeof value === 'number' ? String(value) : (value as any),
+                  })
                 }
+                onChangeOptional={value => onChangeParameterAllow({ ...isAllowedEmptyValues, [parameter.name]: value })}
+                canChangeOptional={true}
+                isOptional={isAllowedEmptyValues[parameter.name] ?? false}
               />
             );
-          }
-
-          return (
-            <ParameterEditor
-              key={parameter.name}
-              parameter={parameter}
-              value={typeof value === 'string' ? value : undefined}
-              onChange={value =>
-                onChangeValues({
-                  ...values,
-                  [parameter.name]: typeof value === 'number' ? String(value) : (value as any),
-                })
-              }
-              onChangeOptional={value => onChangeParameterAllow({ ...isAllowedEmptyValues, [parameter.name]: value })}
-              canChangeOptional={true}
-              isOptional={isAllowedEmptyValues[parameter.name] ?? false}
-            />
-          );
-        })}
+          })}
       </Panel.Content>
     </Panel>
   );
 };
+
+export interface OneOfMenuProps {
+  /** all of the available sub-schemas that can be picked */
+  choices: Choice[];
+
+  /** the currently selected sub-schema */
+  choice: Choice;
+
+  /** called when the selection changes */
+  onChange: (choice: Choice) => void;
+}
+
+/**
+ * When the top level schema is `oneOf`, a drop-down menu that allows the user
+ * to select among the sub-schemas; otherwise `null`.
+ */
+export function OneOfMenu({ choices: subSchemas, choice: Choice, onChange }: OneOfMenuProps) {
```

Comment:
Looks like `Choice` is not used in this component. We could potentially use it for checking selected value in `menuItems`, which might be useful. There's `isChecked` property that we could add.

Reply:
I went ahead and un-renamed that.  Thank you for investigating for me, though.